README.md

```diff
@@ -20,7 +20,11 @@ Daedalus - Cryptocurrency Wallet
 
 [Nix](https://nixos.org/nix/) is needed to run Daedalus in `nix develop` shell.
 
-1. Install nix: `sh <(curl -L https://nixos.org/nix/install)`
+1. Install nix using [Determinate Nix Installer](https://github.com/DeterminateSystems/nix-installer)
+   ```
+   $ curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | \
+   sh -s -- install
```

Comment:
Maybe suggest that the users sets our binary cache from the very beginning? Then it's a single-command setup.

So the command would be:

```
curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | \
  sh -s -- install \
  --extra-conf "substituters = https://cache.nixos.org/ https://cache.iog.io/" \
  --extra-conf "trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ="
```

Reply:
@przemyslaw-wlodek wants to keep the rest of the instruction intact ([Slack](https://input-output-rnd.slack.com/archives/D034MMCM7UG/p1731433623858929)).